### PR TITLE
Add hlint for Haskell

### PR DESCRIPTION
**Pull request by anaynayak**
https://github.com/ndmitchell/hlint

### CODE REVIEW

File: fmts/haskell.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	register(&Fmt{
+		Name: "hlint",
+		Errorformat: []string{
+			`%f:%l:%c: %m`,
+			`%A%f:%l:%c: %.%#: %m`,
+			`%Z%p^%#`,
+			`%C%.%#`,
+			`%-G%.%#`,
+		},
+		Description: "Linter for Haskell soudrce code",
```
Comment (mgrachev) [inline fmts/haskell.go:15]:
```suggestion
		Description: "Linter for Haskell source code",
```

Comment (anaynayak) [reply fmts/haskell.go]: whoops. My bad. I've fixed this now @mgrachev .

**Comment by mgrachev** [comment]
@anaynayak Thank you! 👍

**Comment by review-dog** [comment]
Hi, @anaynayak! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintainance of reviewdog.

Thanks again!

**Pull request merged**